packages/web/src/app/layout.tsx

```
@@ -30,12 +30,13 @@ export const viewport: Viewport = {
 
 export default function Layout({ children }: LayoutProps) {
   return (
-    <html className="h-screen" suppressHydrationWarning>
-      <body className="h-full">
+    <html suppressHydrationWarning>
```

ESLint

[error] <html> elements must have the lang prop.

(jsx-a11y/html-has-lang)